sui/multisig.js

```diff
@@ -153,13 +213,21 @@ if (require.main === module) {
     addBaseOptions(program);
 
     program.addOption(new Option('--txBlockPath <file>', 'path to unsigned tx block'));
-    program.addOption(new Option('--action <action>', 'action').choices(['sign', 'combine', 'execute']).makeOptionMandatory(true));
+    program.addOption(new Option('--action <action>', 'action').choices(['sign', 'combine', 'execute', 'init']).makeOptionMandatory(true));
     program.addOption(new Option('--multisigKey <multisigKey>', 'multisig key').env('MULTISIG_KEY'));
     program.addOption(new Option('--signatures [files...]', 'array of signed transaction files'));
     program.addOption(new Option('--offline', 'run in offline mode'));
     program.addOption(new Option('--combinedSignPath <file>', 'combined signature file path'));
     program.addOption(new Option('--signatureFilePath <file>', 'signed signature will be stored'));
 
+    // The following options are only used with the init action
+    program.addOption(
+        new Option('--base64PublicKeys [base64PublicKeys...]', 'An array of public keys to use for init the multisig address'),
+    );
+    program.addOption(new Option('--weights [weights...]', 'weight for multisig'));
```

Comment:
Mention the default here. Warning isn't needed then

Reply:
The default value for `weights` needs to be an array of 1 which has the same length as `base64PublicKeys`, so i don't think we can set the default value here.